common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -14,14 +22,16 @@
     url = staticfiles_storage.url(file)
 except:
     url = file
-%>${url}${"?raw" if raw else ""}</%def>
+## HTML-escaping must be handled by caller
```

Comment:
Why the requirement that it must be handled by the caller? For example, at the very end of this PR there is a different URL where this assumption is not made.